addons/crm/tests/test_crm_lead_lost.py

```diff
@@ -18,29 +19,96 @@ def setUpClass(cls):
+        self.assertEqual(creation_message.subtype_id, self.env.ref('crm.mt_lead_create'))
+
+        # Update responsible as ACLs is "own only" for user_sales_salesman
         self.lead_1.with_user(self.user_sales_manager).write({
             'user_id': self.user_sales_salesman.id,
             'probability': 32,
         })
+        self.flush_tracking()
 
-        lead = self.lead_1.with_user(self.env.user)
+        lead = self.env['crm.lead'].browse(self.lead_1.ids)
+        self.assertFalse(lead.lost_reason)
         self.assertEqual(lead.probability, 32)
+        self.assertEqual(len(lead.message_ids), 2, 'Should have tracked new responsible')
+        update_message = self.lead_1.message_ids[0]
+        self.assertEqual(update_message.subtype_id, self.env.ref('mail.mt_note'))
 
+        # mark as lost using the wizard
         lost_wizard = self.env['crm.lead.lost'].with_context({
             'active_ids': lead.ids,
         }).create({
-            'lost_reason_id': self.lost_reason.id
+            'lost_reason_id': self.lost_reason.id,
+            'lost_feedback': '<p></p>',  # void content
         })
 
         lost_wizard.action_lost_reason_apply()
+        self.flush_tracking()
 
-        self.assertEqual(lead.probability, 0)
-        self.assertEqual(lead.automated_probability, 0)
+        # check lead update
         self.assertFalse(lead.active)
+        self.assertEqual(lead.automated_probability, 0)
         self.assertEqual(lead.lost_reason, self.lost_reason)  # TDE FIXME: should be called lost_reason_id non didjou
+        self.assertEqual(lead.probability, 0)
+        # check messages
+        self.assertEqual(len(lead.message_ids), 3, 'Should have logged a tracking message for lost lead with reason')
+        update_message = lead.message_ids[0]
+        self.assertEqual(update_message.subtype_id, self.env.ref('crm.mt_lead_lost'))
+        self.assertEqual(len(update_message.tracking_value_ids), 2, 'Tracking: active, lost reason')
+        self.assertTracking(
+            update_message,
+            [('active', 'boolean', True, False),
+             ('lost_reason', 'many2one', False, self.lost_reason)
+            ]
+        )
+
+    @users('user_sales_leads')
+    def test_lead_lost_batch_wfeedback(self):
+        """ Test setting leads as lost in batch using the wizard, including a log
+        message. """
+        leads = self._create_leads_batch(lead_type='lead', count=10, probabilities=[10, 20, 30])
+        self.assertEqual(len(leads), 10)
+        self.flush_tracking()
+
+        lost_wizard = self.env['crm.lead.lost'].with_context({
+            'active_ids': leads.ids,
+        }).create({
+            'lost_reason_id': self.lost_reason.id,
+            'lost_feedback': '<p>I cannot find it. It was in my closet and pouf, disappeared.</p>',
+        })
+        lost_wizard.action_lost_reason_apply()
+        self.flush_tracking()
+
+        for lead in leads:
```

Comment:
Not sure exactly why you create 10 leads?
Is it to test the batching aspect?
Is there not already other tests for that?

Reply:
Indeed, to test batch and ensure it works as intended with active_ids. There are no other tests for lost wizard as far as I know, so indeed this was not done.